src/tools/rbd/action/Qos.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
okey, rbd config for configuration overriding in pool-level and image-level sounds good. So we should use image-meta before that.